distributed/tensor_parallelism/llama2_model.py

```diff
@@ -0,0 +1,462 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
🤔 I would like user to explicitly see the model directly in the examples so that it wouldn't crash/fail if gpt-fast make some changes to the model code thus make the TP API not compatible, so probably maintaining this since it's also quite standalone.